core/variables.js

```diff
@@ -275,19 +275,30 @@ Blockly.Variables.generateUniqueName = function(workspace) {
  */
 Blockly.Variables.createVariableButtonHandler = function(
     workspace, opt_callback, opt_type) {
+  var type = opt_type || '';
   // This function needs to be named so it can be called recursively.
   var promptAndCheckWithAlert = function(defaultName) {
     Blockly.Variables.promptName(Blockly.Msg.NEW_VARIABLE_TITLE, defaultName,
       function(text) {
         if (text) {
-          if (workspace.getVariable(text, opt_type)) {
-            Blockly.alert(Blockly.Msg.VARIABLE_ALREADY_EXISTS.replace('%1',
-                text.toLowerCase()),
+            var existing = Blockly.Variables.nameUsedWithAnyType_(text,
+                workspace);
```

Comment:
`text,` on next line, please

Reply:
Done (moved the whole call to the next line)